.env.sample

```diff
@@ -0,0 +1,6 @@
+POSTGRES_NAME=POSTGRES_NAME
+POSTGRES_USER=POSTGRES_USER
+POSTGRES_PASSWORD=POSTGRES_PASSWORD
+POSTGRES_HOST=POSTGRES_HOST
+POSTGRES_PORT=POSTGRES_PORT
+SECRET_KEY=SECRET_KEY
```

Comment:
no blank line at the end of file, fix everywhere

Reply:
roger that!